src/test/javascript/portal/prototypes/OpenLayersSpec.js

```diff
@@ -177,6 +177,12 @@ describe('OpenLayers', function() {
                 expect(OpenLayers.Geometry.fromWKT('POLYGON((1 2,3 4,1 2))').isBox()).toEqual(false);
             });
         });
+
+        describe('getPrettyBounds', function() {
+            it('returns true when box', function() {
+                expect(OpenLayers.Geometry.fromWKT('POLYGON((1 2,3 2,3 4,1 4,1 2))').getPrettyBounds()).toEqual('1W 2S 3E 4N');
```

Comment:
We should probably test with values with decimal components to ensure that toNSigFigs() is used.